3Delight : Add additional outputs

ericmehl opened the pull request:
This adds a number of missing AOV output presets.

### Checklist ###

- [X] I have read the [contribution guidelines](https://github.com/GafferHQ/gaffer/blob/main/CONTRIBUTING.md).
- [X] I have updated the documentation, if applicable.
- [X] I have tested my change(s) in the test suite, and added new test cases where necessary.
- [X] My code follows the Gaffer project's prevailing coding style and conventions.

johnhaddon commented on startup/gui/outputs.py:
```diff
@@ -169,22 +169,56 @@
-
 		GafferScene.Outputs.registerOutput(
-			"Interactive/3Delight/" + label,
+			"Interactive/3Delight/{}/{}".format( source, displayName ),
```
Can we capitalise `source` here please, so it fits in with the surrounding menu items?

ericmehl replied:
Fixed and squashed into https://github.com/GafferHQ/gaffer/pull/5463/commits/40f4be637a4480ef2d6b41f9171e1453a6fb9f74

johnhaddon commented on startup/gui/outputs.py:
```diff
@@ -195,11 +229,11 @@
 
 		GafferScene.Outputs.registerOutput(
-			"Batch/3Delight/" + label,
+			"Batch/3Delight/{}/{}".format( source, displayName ),
```
Would be good to capitalise source here too I think.

ericmehl replied:
Fixed and squashed into https://github.com/GafferHQ/gaffer/pull/5463/commits/40f4be637a4480ef2d6b41f9171e1453a6fb9f74

johnhaddon reviewed (commented):
Thanks Eric - LGTM apart from some nitpicking about capitalisation.

ericmehl commented:
Great! Fixed up the capitalization and squashed down into the original commit, now https://github.com/GafferHQ/gaffer/pull/5463/commits/40f4be637a4480ef2d6b41f9171e1453a6fb9f74

johnhaddon commented:
Thanks Eric! I've rebased to get the Changes.md update into the right section (we released 1.3.3.0 after the PR was opened). Merging...
